L/LaMEM/build_tarballs.jl

```diff
@@ -73,6 +73,7 @@ platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p
 # MPItrampoline
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
+platforms = filter(p -> !(p["mpi"] == "mpich"), platforms)
```

Comment:
Removing MPICH doesn't sound like a solution, just ignoring a problem

Reply:
A better ***solution***, perhaps, is to come up with an idea of how to pass from here a compat value for MPICH in https://github.com/JuliaPackaging/Yggdrasil/blob/95e2c9f55ef6c82211d039392d7c7febc436c3ea/platforms/mpi.jl#L47 in a backward compatible way so that it doesn't break other existing recipes.

Reply:
Ok that did the job - have a look to see whether you are happy with the implementation.